### PR TITLE
Restart InjectNetworkFrontTest & set up dummy test

### PR DESCRIPTION
**Pull request by holder-josh**
We weren't getting a nice 50/50 sample for the InjectNetworkFrontTest AB test and it's the second time this has happened on the 0% to 2% test sample. 

Therefore:
*We're restarting that test with a different offset and doubled in size
*Set up a dummy test on that 0% to 2% sample to see if the same problem persists

Hopefully by doing this we should be able to definitively say whether there's a problem with the AB testing framework's sampling.

@NataliaLKB do you fancy giving me a quick +1? :sweat_smile:

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/ab.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ define([
     'common/modules/analytics/mvt-cookie',
     'common/modules/experiments/tests/high-commercial-component',
     'common/modules/experiments/tests/membership-message-usa',
-    'common/modules/experiments/tests/inject-network-front-test',
-    'common/modules/experiments/tests/most-popular-default',
```
Comment (NataliaLKB) [inline static/src/javascripts/projects/common/modules/experiments/ab.js]: Why remove the `most-popular-default` test in this file?

**Comment by NataliaLKB** [comment]
Other than the one question, everything else looks good! :+1:

File: static/src/javascripts/projects/common/modules/experiments/ab.js (hunk 1 above)
Comment (Jholder112233) [inline static/src/javascripts/projects/common/modules/experiments/ab.js]: Whoops, good spot. Hasty merge conflict resolving has let me down. Will fix.

Comment (NataliaLKB) [inline static/src/javascripts/projects/common/modules/experiments/ab.js]: Thanks!

**Comment by NataliaLKB** [comment]
:ship:

**Pull request merged**